modules/sievefilters/modules.php

```diff
@@ -359,7 +359,7 @@ public function process() {
             $unblock_sender = false;
             if ($current_script != '') {
                 $base64_obj = str_replace("# ", "", preg_split('#\r?\n#', $current_script, 0)[1]);
-                $blocked_list = json_decode(base64_decode($base64_obj));
+                $blocked_list = json_decode(str_replace("*", "", base64_decode($base64_obj)));
```

Comment:
Do we really want to replace all stars here? Maybe you have an incorrect json syntax because of a star but it is either in the beginning or the end of the script? I think we should be more careful with replacing here - probably use a regex for the exact problem you are trying to solve.